pkg/csi_driver/multishare_controller.go

```diff
@@ -612,7 +613,31 @@ func (m *MultishareController) generateNewMultishareInstance(instanceName string
 	return f, nil
 }
 
-func generateNewShare(name string, parent *file.MultishareInstance, req *csi.CreateVolumeRequest) (*file.Share, error) {
+func (m *MultishareController) checkVolumeContentSource(ctx context.Context, req *csi.CreateVolumeRequest) (string, error) {
+	sourceSnapshotId := ""
+	if req.GetVolumeContentSource() != nil {
+		if req.GetVolumeContentSource().GetVolume() != nil {
+			return "", status.Error(codes.InvalidArgument, "Unsupported volume content source")
```

Comment:
perhaps `Unsupported volume content source type \"volume\"` or something. basically clearer message saying we don't support volume cloning

Reply:
Done.